src/components/Editor/EditorOutline.vue

```diff
@@ -55,9 +55,10 @@ export default {
 <style lang="scss" scoped>
 .editor--outline {
 	width:  300px;
-	padding: 0 10px 10px 10px;
+	padding: 0 10px 100px 10px;
 	position: fixed;
 	overflow: auto;
+	height: calc(100% - 100px);
```

Comment:
This probably still should be `max-height`, to not let the container grow unnecessarily for very short outlines?